### ISSUE TITLE
fine-tune ofa base model for object detection on custom dataset

### ISSUE THREAD

**Issue by ilovecv** [OFA-Sys/OFA#72]
Hi,

Thanks for sharing the awesome work!
Could you please point to me on how to finetune the pretrained ofa model for object detection task? 
Thank you very much!

**Comment by logicwong** [comment]
@ilovecv You can refer to the code of pix2seq (https://github.com/google-research/pix2seq) to fine-tune ofa. We would love to hear results!

**Comment by ilovecv** [comment]
Thanks for the suggestion! Do you have object detection results on MSCOCO?

**Comment by logicwong** [comment]
We haven't tried finetune ofa for object detection.

**Comment by logicwong** [comment]
Sorry for my late reply. Yes, our pretraining datasets include MSCOCO. I'm testing the pretrained model on MSCOCO validation set recently, and I'll let you know as soon as I get the results.

**Comment by ilovecv** [comment]
Thank you very much!

**Comment by logicwong** [comment]
@ilovecv Hi, on MSCOCO val set, the pretrained ofa-large obtains 14.9 MAP score and ofa-base obtains 9.2 MAP score. 
The results are expected, because:

1. OFA used 256 resolution in pre-training, much smaller than 1333 resolution used by pix2seq
2. OFA does not use the data augmentation technique of pix2seq during pre-training
3. The detection datasets contain OpenImages, Object365, etc., and COCO only accounts for a small part of it. Therefore, OFA may generate labels that are out of the COCO label set during inference.

**Comment by ilovecv** [comment]
@logicwong Thanks for your reply. The information is very helpful.

**Comment by daoyuan98** [comment]
> 

Thank you so much for your information! May I check the reported results are obtained by directly evaluating OFA on COCO, instead of finetuning on it? Thank you!

**Comment by JulioZhao97** [comment]
> 

hi, could you please provide the evaluation code of OFA in object detection? I test on COCO val set but only get mAP=1.5. Thank you very much!

**Comment by sunnie-star** [comment]
> > 
> 
> hi, could you please provide the evaluation code of OFA in object detection? I test on COCO val set but only get mAP=1.5. Thank you very much!



Hello,~ May I ask if you have any progress on the code for testing object detection performance using OFA pre trained models? Thank you very much~
